app/src/main/java/de/tum/in/tumcampusapp/component/tumui/lectures/activity/LecturesDetailsActivity.java

```diff
@@ -84,45 +64,66 @@ public void onCreate(Bundle savedInstanceState) {
         tvLDetailsZiele = findViewById(R.id.tvLDetailsZiele);
         tvLDetailsTermin = findViewById(R.id.tvLDetailsTermin);
         tvLDetailsLiteratur = findViewById(R.id.tvLDetailsLiteratur);
+
         btnLDetailsTermine = findViewById(R.id.btnLDetailsTermine);
-        btnLDetailsTermine.setOnClickListener(this);
+        btnLDetailsTermine.setOnClickListener(v -> {
+            Bundle bundle = new Bundle();
+
+            // LectureAppointments need the name and id of the facing lecture
+            bundle.putString("stp_sp_nr", currentItem.getStp_sp_nr());
+            bundle.putString(Const.TITLE_EXTRA, currentItem.getTitle());
+
+            Intent i = new Intent(this, LecturesAppointmentsActivity.class);
+            i.putExtras(bundle);
+            startActivity(i);
+        });
+
+        mLectureId = getIntent().getStringExtra("stp_sp_nr");
```

Comment:
that string needs to be extracted to a common constant

Reply:
Agreed. Can you (or @kordianbruck) tell me what this abbreviation stands for?